sbin/pfdhcplistener

```diff
@@ -198,13 +191,25 @@ sub process_pkt {
         my $l2 = NetPacket::Ethernet->decode($pkt);
```

Comment:
Because the there is a different action for dhcpv6 and dhcp which will most likely lead to 2 different queues